### ISSUE TITLE
Add compatibility to TYPO3 Ver. 9 LTS

### ISSUE THREAD

**Issue by tschikarski** [tschikarski/shibboleth#29]

**Comment by tschikarski** [comment]
This might be helpful: https://github.com/Tuurlijk/typo3scan

**Comment by cnoelker** [comment]
Are there any plans to update the extension?

**Comment by tschikarski** [comment]
Yes, this plan exists. However, I cannot tell exactly, when this can be done. I will take care about a clarification next week.

**Comment by martinhaase** [comment]
Hi, here's a +1 for Typo3 v9. Any news on this? Seems like everybody has v9 meanwhile...

**Comment by sneopu** [comment]
I fixed this extension for a customer to run on TYPO3 v10. Anyone interested?
If there is a need, I will invest some more work and share it.

**Comment by Hobokens** [comment]
Hi @sneopu 
Thanks for your offer. I'm very interested :-)
Regards
Hobokens

**Comment by sneopu** [comment]
@Hobokens I made a fork and commited my code which should work with v10. But do some proper testing.
https://github.com/sneopu/shibboleth/tree/TYPO3-v10-dev

**Comment by Hobokens** [comment]
@sneopu thank you very much for sharing your work. 
We are currently planning to set up an intranet for our university. In this context, I will test the extension and report back.
